test/extended/testdata/test-cli-debug.yaml

```diff
@@ -0,0 +1,69 @@
+kind: List
+apiVersion: v1
+items:
+- kind: ImageStream
+  apiVersion: v1
+  metadata:
+    name: busybox
+  spec:
+    tags:
+    - name: latest
+      from:
+        name: busybox
+        kind: DockerImage
+
+- kind: DeploymentConfig
+  apiVersion: v1
+  metadata:
+    name: busybox1
+  spec:
+    replicas: 0
+    selector:
+      deploymentconfig: busybox1
+    template:
+      metadata:
+        labels:
+          deploymentconfig: busybox1
+      spec:
+        containers:
+        - name: busybox
```

Comment:
@bparees I don't believe this is true.  My understanding is that the container name must be specified; if the image name is omitted (which it is), the deployment is forced to use the image from the imagechangetrigger.

My code was based on the assumption that it was permissible to attempt to get from a docker image to an imagestream name by removing the leading w.x.y.z:5000/namespace/ .  This assumption was sufficient to match previous functionality and for the test to pass.